### ISSUE TITLE
Version information on projects that use nom.

### ISSUE THREAD

**Issue by wmealing** [Geal/nom#1089]
Hello, 

I was wondering if this would be a welcome PR. The readme.md mentions a list of "Parsers written with nom" These are written with many different versions of nom.  I (as a user) would find it useful if I could know which parsers are written in which nom release so that I can browse the projects to learn from them.

I want to modify readme.md , mention the versions specified from the projects cargo.toml that each of the projects use and submit a pull request in a branch.

Like this

Text file formats:
   Ceph Crush (nom = "^2.0")
   Cronenberg (nom = "4.0.0-alpha2")

I thought I'd ask before I go to the effort.

Thanks in advance.

**Comment by Geal** [comment]
right, that would be useful (especially when people want examples for specific versions)
